pkg/cvo/sync_worker.go

```diff
@@ -1184,6 +1191,8 @@ func condenseClusterOperators(errs []error) []error {
 			case payload.UpdateEffectNone:
 			case payload.UpdateEffectFail:
 				updateEffect = payload.UpdateEffectFail
+			case payload.UpdateEffectReport:
+				updateEffect = payload.UpdateEffectReport
```

Comment:
In order of increasing severity:

* `None`
* `Report`
* `FailAfterInterval`
* `Fail`

So we should have a guard here like we already have for `FailAfterInterval` that says "if we have an error like this, and the aggregated `updateEffect` is not already worse, set the aggregated `updateEffect`:

```go
case payload.UpdateEffectReport:
  if updateEffect != payload.UpdateEffectFail && updateEffect != payload.UpdateEffectFailAfterInterval {
    updateEffect = payload.UpdateEffectReport
  }
```

Reply:
Hrm, or maybe `None` is more severe than `Report`?  It's hard to rank one effect that blocks reconciliation completion but does not get inserted into `Failing` (`None`) vs. another that does not block reconciliation completion but does get inserted into `Failing` (`Report`).  We might need to bump `condenseClusterOperators` to return `(blocking, report error)` or something.

Reply:
Made `UpdateEffectReport` least severity.